<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnnealedParticleFilterHistoryWidgetBase</class>
 <widget class="QWidget" name="AnnealedParticleFilterHistoryWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>particles</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QTreeWidget" name="particleTreeWidget">
        <column>
         <property name="text">
          <string notr="true">1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="horizontalSpacing">
       <number>3</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QToolButton" name="setPoseButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/wombat.qrc">
            <normaloff>:/icons/24/view-refresh.png</normaloff>:/icons/24/view-refresh.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="copyPoseButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/wombat.qrc">
            <normaloff>:/icons/24/edit-copy.png</normaloff>:/icons/24/edit-copy.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="saveResultsButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/wombat.qrc">
            <normaloff>:/icons/24/document-save-as.png</normaloff>:/icons/24/document-save-as.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="findOptimumButton">
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../resources/wombat.qrc">
            <normaloff>:/icons/24/system-search.png</normaloff>:/icons/24/system-search.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>particle details</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="particleLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/wombat.qrc"/>
 </resources>
 <connections/>
</ui>
